CHANGELOG_PENDING.md

```diff
@@ -45,6 +45,7 @@ program](https://hackerone.com/tendermint).
 - Go API
   - [libs/pubsub] [\#4070](https://github.com/tendermint/tendermint/pull/4070) `Query#(Matches|Conditions)` returns an error.
   - [rpc/client] \#3471 `Validators` now requires two more args: `page` and `perPage`
+  - [libs/common] \#3262 Make error the last parameter of `Task` (@PSalant726)
```

Comment:
also cryptoAmino rename.

Also I thought I saw some `XYZError` get renamed to `ErrXYZ` but I can't find it now and not sure it was even in a package covered by our versioning

Reply:
> also cryptoAmino rename.

how's this breaking? I am not even sure it's worth mentioning in the changelog

Reply:
isn't it a change in package name?

Reply:
imo it's only breaking if you rename the folder

```
Client code uses the package path when importing the package. By convention, the last element of the package path is the package name
```
https://blog.golang.org/package-names